starter/app/src/main/java/com/example/android/politicalpreparedness/election/VoterInfoFragment.kt

```diff
@@ -1,32 +1,54 @@
 package com.example.android.politicalpreparedness.election
 
+import android.content.Intent
+import android.net.Uri
 import android.os.Bundle
 import android.view.*
```

Comment:
do not use wildcards

<img width="985" alt="image" src="https://user-images.githubusercontent.com/1131857/172497349-73a6b689-c12b-4036-8b74-a2abb10be88e.png">

Reply:
use CMD+OPTION+O